server/src/metadata/data-source/data-source.service.ts

```diff
@@ -121,6 +121,20 @@ export class DataSourceService implements OnModuleInit, OnModuleDestroy {
     return `workspace_${uuidToBase36(workspaceId)}`;
   }
 
+  public async deleteWorkspaceSchema(workspaceId: string) {
+    const schemaName = this.getSchemaName(workspaceId);
```

Comment:
Note for later: I think this is fine for now but ideally we should fetch the schemaName from the datasource-metadata which should be the source of truth.